core/src/main/java/org/apache/brooklyn/util/core/osgi/Osgis.java

```diff
@@ -90,46 +69,8 @@
     private static final Logger LOG = LoggerFactory.getLogger(Osgis.class);
 
     private static final String EXTENSION_PROTOCOL = "system";
-    private static final String MANIFEST_PATH = "META-INF/MANIFEST.MF";
     private static final Set<String> SYSTEM_BUNDLES = MutableSet.of();
 
-    public static class VersionedName {
```

Comment:
violates deprecation policy -- mark as `/** @deprecated since 0.9.0, replaced with top-level class */`.  this impl could extend the top-level class.  that will ensure code using this does not break.